README.md

```diff
@@ -50,112 +42,41 @@ $ pip install dagster dagit
 
 This installs two modules:
 
-- **Dagster**: the core programming model and abstraction stack; stateless, single-node,
-  single-process and multi-process execution engines; and a CLI tool for driving those engines.
-- **Dagit**: the UI for developing and operating Dagster pipelines, including a DAG browser, a
-  type-aware config editor, and a live execution interface.
+- **Dagster**: The core programming model.
+- **Dagit**: The web interface for developing and operating Dagster jobs. It includes a DAG browser,
+a type-aware interface to launch runs, a live view for in-progress runs, a catalog to view your data
+assets, and more.
 
-### Learn
+For a quick overview, check out our [Getting Started](https://docs.dagster.io/getting-started) page.
```

Comment:
I think we should have an "Example" or "Quick Start" section here to show code snippets as opposed to redirecting folks to docs site - cc @sryza / @erinkcochran87 do you have thoughts on what code to include as the starter code? - we can default to the same thing as the [Getting Started](https://docs.dagster.io/getting-started#writing-a-job) for now and switch to assets content later.

Reply:
If we went the route of it actually being a quick start, I could be on board with it. I feel like including examples would require additional explanation that's typically too lengthy for a README.

I think it'd also be cool to add something that's similar to what [Jekyll does here](https://github.com/jekyll/jekyll#getting-started), specifically the **Getting started** and **Diving in** sections. After someone does the quick start, I think pointing them in the direction of next steps would be helpful. (FWIW, I also think we should do this in the docs - working on that 😉  )

Reply:
I like the idea of having **Getting started** and **Diving in** sections. Look forward to the changes in the docs.